ads/3p.md

```diff
@@ -0,0 +1,99 @@
+
+
+In order to ensure minimum latency and quality, please follow the instructions listed here before submitting a pull request to the AMP open source project. For general guidance on how get started with contributing to the AMP project, please see here.
+
+##### Side Platform (SSP) or an Ad Exchange
+
+*Examples : Rubicon, Criteo OR Appnexus, Ad-Exchange*
+
+If you are a sell side platform that wants to directly get called from a publisher’s webpage, you will need to follow the same directions as listed above for integrating with an Ad Server. Adding your own ‘type’ value to the amp-ad tag will allow you to distribute your tag directly to the publisher, so they can insert your tags directly into their AMP pages.
+
+More commonly, SSPs work with the publisher to traffick the SSP’s ad tags in their ad server. In this case, ensure that all assets being loaded by your script in the ad server’s creative are being made over HTTPS. There are some restrictions around some intrusive ad formats like expandables, so we recommend that you test out the most commonly delivered creative formats with your publishers.
+
+#### Ad Agency
+*Examples : Essence, Omnicom*
+
+Work with your publisher to ensure that the creatives you develop are AMP compliant. Since all creatives are served into iframes whose size is determined, when the ad is called, ensure that your creative doesn't try to modify the size of the iframe.
+
+Ensure that all assets that are part of the creative are requested using HTTPS.
+Some ad formats are fully not supported at the moment and we recommend testing the creatives in an AMP environment. Some examples are : Rich Media Expandables, Interstitials, Page Level Ads
+
+#### Video Player
+
+*Examples : Brightcove, Ooyala*
+
+A video player that works in regular HTML pages will not work in AMP and therefore a specific tag needs to be created that will allow the AMP runtime to load your player. 
+Brightcove has already created a custom amp-brightcove tag that allows media and ads to be played in AMP pages.
+
+A brightcove player can be invoked by the following:
+```
+<amp-brightcove
+      data-account="906043040001"
+      data-video-id="1401169490001"
+      data-player-id="180a5658-8be8-4f33-8eba-d562ab41b40c"
+      layout="responsive" width="480" height="270">
+  </amp-brightcove>
+```
+For instructions on how to develop an amp tag like brightcove, please see [amp-brightcove.js](https://github.com/ampproject/amphtml/blob/master/extensions/amp-brightcove/0.1/amp-brightcove.js)
+
+The other popular option is for video players to take advantage of the amp-iframe (https://github.com/ampproject/amphtml/blob/master/extensions/amp-iframe/amp-iframe.md) extension (with post frame or fallback markup) and serve the player within the iframe. 
```

Comment:
Its a good point that we could steer them to `amp-embed` which may be more appropriate.

I think you are right, that for 3p platforms (as opposed to, say, ESPN's video player) we should not recommend amp-iframe. I originally suggested this, but lets drop.